### PR TITLE
More instructions for Homebrew users

### PR DESCRIPTION
**Pull request by amontoison**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -41,13 +41,18 @@ It supports real and complex, single or double precision arithmetic.
 
 **Note: qr_mumps is already precompiled with Yggdrasil for all platforms.**
 
-To use your custom qr_mumps, set the environmental variables `JULIA_QR_MUMPS_LIBRARY_PATH`
-to point the shared library. Note that **qr\_mumps** version 3.0.3 is needed.
+To use your custom qr_mumps, set the environmental variables `JULIA_QRMUMPS_LIBRARY_PATH`
+to point to the shared library before `using QRMumps`.
```
Comment (dpo) [inline README.md:45]:
```suggestion
To use your custom qr_mumps, set the environment variable `JULIA_QRMUMPS_LIBRARY_PATH`
to point to the folder holding the qr_mumps shared libraries before `using QRMumps`.
```

File: README.md [hunk 2]
```diff
@@ -41,13 +41,18 @@ It supports real and complex, single or double precision arithmetic.
...
+Note that the same version of qr_mumps as used by the `qr_mumps_jll` artifact is needed.
+To initialize qr_mumps with a custom installation, the function [`qrm_init`](https://juliasmoothoptimizers.github.io/QRMumps.jl/stable/api/#QRMumps.qrm_init) must be called prior to any other functions of QRMumps.jl.
```
Comment (dpo) [inline README.md:47]: Why isn't it possible to define
```julia
  function __init__()
    qrm_init()
  end
```
as you do for the artifact?

Comment (amontoison) [reply README.md:47]: We can also use the `__init__()` function for custom installations but it locks the number of cpu threads to 1 and the number of gpu threads to 0.
If the user uses a custom installation, I suppose that he wants to use the features not supported by the artifact.

Comment (dpo) [reply README.md:47]: Oh I see. I did not realize we could not set the number of CPUs when using the artifact.

**Pull request merged**